packs/equipment/cold-iron-buckler-high-grade.json

```diff
@@ -0,0 +1,54 @@
+{
+    "_id": "lit8IyoUHuybenvU",
+    "img": "systems/pf2e/icons/equipment/shields/precious-material-shields/cold-Iron-buckler.webp",
+    "name": "Cold Iron Buckler (High-Grade)",
```

Comment:
I'm not sure we want to add these items in, but I see that the links in the existing roll tables reference non-existent items, and therefore the links are broken. Any thoughts @TikaelSol?

Reply:
We don't want to add them. We removed them years ago

Reply:
Should I remove the entries from the table or make them text entries? And should I assume that any other entries that don't have corresponding items in the compendium should be handled the same way?

Reply:
Maybe link to the base item?

Reply:
I believe the base item for all  is **Cold Iron Shield** GM Core p233, unless I am mistaken.

Reply:
I was thinking of linking to Buckler, for example.

Reply:
Would Steel Shield be the base item for shields? (I swear I'm not trying to be difficult)

Reply:
Yeah, that sounds reasonable.

Reply:
Dragging in buckler then editing the result details to match the books' tables breaks the item. They'll have to be text entries.